recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "optuna" %}
-{% set version = "3.5.0" %}
+{% set version = "3.6.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: ca9e1ce16aa6c6a5af0e1cc1d0cbcd98eb1c75b6a2f06be6bd9c0c5ab0698724
+  sha256: 0f023303d9212916e41dce960ed4bcdf3034daac4c58160bc9262ea473484c9c
```

Comment:
This is consistent with https://pypi.org/project/optuna/3.6.0/#copy-hash-modal-44bbf9eb-e672-44b1-af73-f7984d266a96